src/VS.Web.CG.Mvc/Templates/Identity/Bootstrap4/Pages/_Layout.cshtml

```diff
@@ -25,59 +25,62 @@
-                {
-                    throw new InvalidOperationException("The default Identity UI layout requires a partial view '_LoginPartial' " +
-                        "usually located at '/Pages/_LoginPartial' or at '/Views/Shared/_LoginPartial' to work. Based on your configuration " +
-                        $"we have looked at it in the following locations: {System.Environment.NewLine}{string.Join(System.Environment.NewLine, result.SearchedLocations)}.");
-                }
+                <div class="navbar-collapse collapse d-sm-inline-flex flex-sm-row-reverse">
+                    @@{
+                        var result = Engine.FindView(ViewContext, "_LoginPartial", isMainPage: false);
+                    }
+                    @@if (result.Success)
+                    {
+                        await Html.RenderPartialAsync("_LoginPartial");
+                    }
+                    else
+                    {
+                        throw new InvalidOperationException("The default Identity UI layout requires a partial view '_LoginPartial' " +
+                            "usually located at '/Pages/_LoginPartial' or at '/Views/Shared/_LoginPartial' to work. Based on your configuration " +
+                            $"we have looked at it in the following locations: {System.Environment.NewLine}{string.Join(System.Environment.NewLine, result.SearchedLocations)}.");
+                    }
+                </div>
             </div>
-        </div>
-    </nav>
-    <div class="container body-content">
-        @@RenderBody()
-        <hr />
-        <footer>
-            <p>&copy; @currentYear - @Model.ApplicationName</p>
-        </footer>
+        </nav>
+    </header>
+
+    <div class="container">
+        <partial name="_CookieConsentPartial" optional />
+        <main role="main" class="pb-1">
+            @@RenderBody()
+        </main>
     </div>
+    <footer class="footer border-top pl-3 text-muted">
+        <div class="container">
+            &copy; @currentYear - @Model.ApplicationName - <a asp-area="" asp-page="Privacy">Privacy</a>
+        </div>
+    </footer>
 
     <environment include="Development">
         <script src="~/Identity/lib/jquery/dist/jquery.js"></script>
-        <script src="~/Identity/lib/bootstrap/dist/js/bootstrap.js"></script>
+        <script src="~/Identity/lib/bootstrap/dist/js/bootstrap.bundle.js"></script>
         <script src="~/Identity/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.3.1.min.js"
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"
                 asp-fallback-src="~/Identity/lib/jquery/dist/jquery.min.js"
                 asp-fallback-test="window.jQuery"
                 crossorigin="anonymous"
                 integrity="sha384-tsQFqpEReu7ZLhBV2VZlAu7zcOV+rXbYlF2cqB8txI/8aZajjp4Bqd+V6D5IgvKT">
```

Comment:
Nitpick, but in aspnet/Templating we use sha256 since that seems to be the "normal" sha for SRI.

Reply:
@natemcmaster may be able to address this, sha's were updated here #795 
But it appears we've been using sha384 in scaffolding for quite a while (since before I was part of it).

Reply:
SHA-256, SHA-384 and SHA-512 are all supported. The default I've seen in most docs and specs is sha384.